docker-compose.yml

```diff
@@ -50,6 +50,8 @@ services:
       - indexer
       - rabbitmq
       - redis
+    depends_on:
+      - worker
```

Comment:
@mihaibivol tentative to solve an error that was caused buy something else. Actually indeed a leftover. Well spot.